collection-spec/collection-spec.md

```diff
@@ -23,19 +23,19 @@ Implementations are encouraged, however, as good effort will be made to not chan
 
 ## Collection fields
 
-| Element      | Type              | Description                                                  |
-| ------------ | ----------------- | ------------------------------------------------------------ |
-| stac_version | string            | **REQUIRED.** The STAC version the Collection implements. STAC versions can be mixed, but please keep the [recommended best practices](../best-practices.md#mixing-stac-versions) in mind. |
-| stac_extensions | [string]       | A list of extensions the Collection implements. |
-| id           | string            | **REQUIRED.** Identifier for the collection that is unique across the provider. |
-| title        | string            | A short descriptive one-line title for the collection.       |
-| description  | string            | **REQUIRED.** Detailed multi-line description to fully explain the collection. [CommonMark 0.29](http://commonmark.org/) syntax MAY be used for rich text representation. |
-| keywords     | [string]          | List of keywords describing the collection.                  |
-| license      | string            | **REQUIRED.** Collection's license(s), either a SPDX [License identifier](https://spdx.org/licenses/), `various` if multiple licenses apply or `proprietary` for all other cases. |
-| providers    | [[Provider Object](#provider-object)] | A list of providers, which may include all organizations capturing or processing the data or the hosting provider. Providers should be listed in chronological order with the most recent provider being the last element of the list. |
-| extent       | [Extent Object](#extent-object) | **REQUIRED.** Spatial and temporal extents.    |
-| summaries    | Map<string, [*]\|[Stats Object](#stats-object)> | A map of property summaries, either a set of values or statistics such as a range. |
```

Comment:
Something funny happened to the formatting on this line

Reply:
That's the added space in `Map<string, [*]\  |[Stats Object](#stats-object)>`. I think Josh thought it's a table separator, but it was only an "or" (i.e. an escaped `|`) and thus corrupts the table.

Reply:
I actually went back and tried to make some of the tables a little cleaner.  In this instance, it just so happens, when looking at the raw markdown, that the pipe lines up with the other table separator pipes.  However it's mere coincidence -- I simply didn't make every row have the same fixed-width for the Type field because there are only 3 lines that are much longer than all of the others.  I'll just go ahead and extend them all out so it looks cleaner.

Reply:
Or maybe I did introduce new spaces before the "or"?  Lol not sure, but I'll fix it up either way.

Reply:
It was the latter, see the fix below :-)